lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -123,7 +123,12 @@ Function Join-Domain {
 
     if($domain_ou_path){
         Write-DebugLog "adding OU destination arg to Add-Computer args"
-        $add_args["OUPath"] = $domain_ou_path
+        if([System.DirectoryServices.DirectoryEntry]::Exists("LDPA://$domain_ou_path")){
```

Comment:
Wouldn't this be `LDAP` instead of `LDPA`?

Reply:
Yes, you are right - that would be better, I'll fix it

Reply:
I'm also wondering whether we can use the `Get-ADObject` cmdlet instead. It would keep things more PowerShell like and can handle all the LDAP details inside it. What are your thoughts?

Reply:
Yes, we can also use this; I'll check the syntax and push some change

Reply:
But the issue here is, that this will only work on Server Systems and not on Windows Client OS, but the System.DirectoryServices Namespaces, is as well on clients as on servers